app/seed-data/emissions_factors/scripts/EFDB_2006_IPCC_guidelines_Stationary_Energy.py

```diff
@@ -0,0 +1,631 @@
+
+    # Function to expand metadata dictionary into separate columns
+    def expand_metadata(metadata):
+        return pd.Series(metadata)
+
+    # Apply conditions and update the metadata column
+    conditions = ['Net Calorific Value', 'NCV: ', 'density:']
+    for condition in conditions:
+        mask = EF_df['properties'].str.contains(condition, na=False)
+        EF_df.loc[mask, ['NCV_value', 'NCV_units', 'density_value', 'density_units']] = EF_df.loc[mask, 'properties'].apply(lambda x: expand_metadata(extract_metadata(x, condition))) 
+
+    # drop extra columns
+    EF_df = EF_df.drop(columns=['value_min', 'value_max'])
+
+    # filter the first df to extract Net Calorific Values to transform EF into other units
+    # extract GCV and NCV
+    filt_NCV = df["Description"].str.contains('NCV', case=False, na=False)
+    filt_df = df.loc[filt_NCV].reset_index(drop=True)
+
+    # drop extra columns
+    filt_df = filt_df.drop(columns=['IPCC 1996 Source/Sink Category', 'Fuel 1996', 'C pool', 'Type of parameter', 'Abatement / Control Technologies', 'IPCC Worksheet', 'Source of data', 'Data provider'])
+
+    # New list to hold NCV and GCV values
+    tmp = []
+
+    # Process each row
+    for index, row in filt_df.iterrows():
+        value = str(row['Value'])
+        
+        if '(GCV)' in value and '(NCV)' in value:
+            # Extract the values
+            parts = value.split('(GCV)')
+            gcv_value = parts[0].strip()
+            ncv_value = parts[1].split('(NCV)')[0].strip()
+            # Create new rows for GCV and NCV values
+            gcv_row = row.copy()
+            gcv_row['Value'] = gcv_value
+            gcv_row['Description'] = 'Gross Calorific Value'
+            
+            ncv_row = row.copy()
+            ncv_row['Value'] = ncv_value
+            ncv_row['Description'] = 'Net Calorific Value'
+            
+            # Append new rows
+            tmp.append(gcv_row)
+            tmp.append(ncv_row)
+        else:
+            tmp.append(row)
+
+    # Convert the data back to a DataFrame
+    filt_df = pd.DataFrame(tmp)
+
+    # standardization of units
+    for index, row in filt_df.iterrows():
+        units = row['Unit']
+        value = row['Value']
+        
+        if 'GJ/1000m3' in units:
+            filt_df.at[index, 'Value'] = np.float16(value) * 1e-3
+            filt_df.at[index, 'Unit'] = 'GJ/m3'
+            
+        elif units in ['GJ/1000 litre', 'GJ/1000litre']:
+            filt_df.at[index, 'Value'] = np.float16(value) * 1e-3
+            filt_df.at[index, 'Unit'] = 'GJ/litre'
+            
+        elif units == 'TJ/10^6m3 @SATP':
+            filt_df.at[index, 'Value'] = np.float16(value) * 1e-6
+            filt_df.at[index, 'Unit'] = 'TJ/m3'
+            
+        elif units == 'TJ/10^3kl @SATP':
+            filt_df.at[index, 'Value'] = np.float16(value) * 1e-3
+            filt_df.at[index, 'Unit'] = 'TJ/kl'
```

Comment:
A case that prints an error message if a unit was not recognized would be helpful here. So we know we have do add more clauses for different unit conversions when other units are added in the future.

```suggestion
        elif units == 'TJ/10^3kl @SATP':
            filt_df.at[index, 'Value'] = np.float16(value) * 1e-3
            filt_df.at[index, 'Unit'] = 'TJ/kl'
        else:
            print("Error: unrecognized units - ", units, " - line ", index)
```